pkg/scalers/kafka_scaler.go

```diff
@@ -191,6 +192,16 @@ func parseKafkaMetadata(config *ScalerConfig) (kafkaMetadata, error) {
 		meta.allowIdleConsumers = t
 	}
 
+	meta.version = sarama.DefaultVersion
```

Comment:
Should we depend here on the Sarama's default version (which could change in the future?) or hardcoded this to the default version used by KEDA (ie. `sarama.V1_0_0_0`)? 🤷‍♂️

Reply:
In the docs PR, there is mentioned that default is 1.0.0 not sarama default. 

I might be just nit picking, so would love to hear some feedback on this.

Reply:
Hello,
Yes indeed you are right, it could potentially lead to a breaking change if we upgrade sarama lib and don't check this variable.
I can change it to 1.0.0 to fix it if you are ok with it :)

Reply:
@tomkerkhove PTAL

Reply:
Yeah, please change the code to 1.0.0 :)

Reply:
Done